source/runbooks/dr-process.html.md.erb

```diff
@@ -0,0 +1,362 @@
+
+**Core Accounts** - Located [here](https://github.com/ministryofjustice/modernisation-platform/actions), on the left you will see the deployment workflows for all core based accounts, required for the building on the platform.
+
+However, I would run manually just to monitor the plan, help troubleshoot issues, as sometimes these scenarios are not as cut and dry as all infrastructure has vanished from one account.
+
+
+#### Manually
+
+
+1.) Run a terraform plan against the production workspace, in modernisation-platform .
+
+2.) All changes should appear, using your admin credentials, run an apply.
+
+3.) Assuming the apply runs as planned, login to the account and check that vpcs, etc look correct.
+
+4.) In modernisation-platform-environments , run terraform against the correct account.
+
+
+
+### Things to remember
+
+
+- Statefile for the lost account will exist, this might need to be removed.
+- Any backups on the account or volumes will be lost.
+
+
+## Disaster Recovery Running outside of London.
+
+
+If the London infrastructure is to be down for the foreseeable future, running from another region is the next step.
+
+
+
+### Informing teams
+
+
+Firstly, inform the following channels of the situation and put forward a contactable member of staff from our team.
+
+
+`#ask-modernisation-platform`
+
+
+`#modernisation-platform`
+
+
+`#modernisation-platform-alerts`
+
+
+Supply the ETAs, lead incident numbers, links to where updates will be posted etc. We will mention our commitment to regularly we will update the incident, however until we know what is expected this is an unknown.
+
+
+### Cross Region Infrastructure in the Platform
+
+
+| Technology | Information | Replicated? | Replication Region |
+|----------------- |------------------------------------------------ |------------- |-------------------- |
+| S3 | Contains the platforms terraform state files. | Yes | eu-west-1 |
+| KMS | Contains encryption keys for the platform. | No | - |
+| Secrets Manger | Contains platform secrets. | Yes | eu-west-1 |
+| IAM Policies | Across all regions. | - | - |
+| DNS | Across all regions. | - | - |
+| VPC | No VPCs currently exist for DR, only reserved. | - | - |
+| Firewall | Not cross region. | | |
+| Transit Gateway | Not cross region. | | |
+| NACLS | Not cross region. | | |
+| Subnets | Not cross region. | | |
+| Dynamo Disable | Check
+
+
+
+### Repo / Module List
+
+
+These are the repositories owned by the Modernisation Platform.
+
+
+The next step will be to create new tags for the current releases with any hardcoded region values changed.
+
+
+`0.0.0`
+
+
+Use the version number stated so these changes are easily recognisable. The version of these modules will then need to be updated in our
+
+
+`Modernisation Platform` repository, along with any variables changed that are needed to. This should go in one large PR with many reviewers.
```

Comment:
One large PR is when things usually go wrong. I think breaking it into smaller step by step, easily revertible PRs is what we are after.

Reply:
Good point, there will be obvious dependencies such as VPC and subnets, transit GTW, route tables, etc. but you could group the resources and apply only one area at the time (e.g. IAM roles, networking, logging, security baseline)